### PR TITLE
Teams Tweaks

### PR DESCRIPTION
**Pull request by JoshOldenburg**
Filter the kiosk to only show visible and attendable teams, filter the attendance admin page to only show attendable teams, and stop refreshing the team edit page every time it is submitted.

### CODE REVIEW

File: resources/assets/js/components/teams/TeamEditForm.vue [hunk 1]
```diff
@@ -216,7 +216,8 @@ export default {
           this.hasError = false;
           this.feedback = 'Saved!';
           console.log('success');
-          window.location.href = '/admin/teams/' + response.data.team.slug;
+          var newHREF = '/admin/teams/' + response.data.team.slug;
+          if (!window.location.href.endsWith(newHREF)) window.location.href = newHREF;
```
Comment (kberzinch) [inline resources/assets/js/components/teams/TeamEditForm.vue]: Why is the reload necessary?

File: resources/assets/js/components/attendance/AttendanceKiosk.vue [hunk 2]
```diff
@@ -43,7 +43,9 @@ export default {
           if (rawTeams.length < 1) {
             swal('Bueller...Bueller...', 'No teams found.', 'warning');
           } else {
-            this.teams = response.data.teams.sort(function(a, b) {
+            this.teams = response.data.teams.filter(function(item) {
+              return item.visible && item.attendable;
```
Comment (kberzinch) [inline resources/assets/js/components/attendance/AttendanceKiosk.vue]: Only check for `attendable` - we may want to have teams that appear on the kiosk but not on the website (training maybe???)

File: resources/assets/js/components/teams/TeamEditForm.vue (hunk 1 above)
Comment (JoshOldenburg) [reply resources/assets/js/components/teams/TeamEditForm.vue]: That form is also used for creating the team initially, so it'll redirect from `/admin/teams/create` to `/admin/teams/<slug>`. When editing an existing one it won't redirect now.

Comment (kberzinch) [reply resources/assets/js/components/teams/TeamEditForm.vue]: Oh derp this is on the edit form, carry on.

File: resources/assets/js/components/attendance/AttendanceKiosk.vue (hunk 2 above)
Comment (kberzinch) [reply resources/assets/js/components/attendance/AttendanceKiosk.vue]: Training definitely per bossman

File: resources/assets/js/components/attendance/AttendanceKiosk.vue [hunk 3]
```diff
@@ -43,7 +43,9 @@ export default {
           if (rawTeams.length < 1) {
             swal('Bueller...Bueller...', 'No teams found.', 'warning');
           } else {
-            this.teams = response.data.teams.sort(function(a, b) {
+            this.teams = response.data.teams.filter(function(item) {
+              return item.attendable;
```
Comment (kberzinch) [inline resources/assets/js/components/attendance/AttendanceKiosk.vue]: Switch back to attendable and visible

**Pull request merged**